src/System.Runtime.Extensions/src/System/IO/BufferedStream.cs

```diff
@@ -50,23 +49,23 @@ public sealed class BufferedStream : Stream
         private const int MaxShadowBufferSize = 81920;  // Make sure not to get to the Large Object Heap.
         private const int DefaultBufferSize = 4096;
 
-        private Stream _stream;                             // Underlying stream.  Close sets _stream to null.
-        private byte[] _buffer;                             // Shared read/write buffer.  Alloc on first use.
+        private Stream? _stream;                             // Underlying stream.  Close sets _stream to null.
```

Comment:
nit: should this be non-nullable considering it's only non-null after Close?

Reply:
I don't think so.  Plus it's exposed out of BufferedStream.UnderlyingStream, which doesn't do any validation on the stream having been closed, and thus can return null after Dispose.